design-documents/graph-ql/coverage/catalog/catalog-product-price.graphqls

```diff
@@ -0,0 +1,21 @@
+type ProductPriceRange {
+    minimum_price: ProductPrice!
+    maximum_price: ProductPrice
+}
+
+type ProductPrice {
+    regular_price: Money!
```

Comment:
I just want to add that `Money!` is a really funny name for a type.

Reply:
slightly enterprise-ish https://martinfowler.com/eaaCatalog/money.html